.cargo/config.toml

```
@@ -1,4 +1,4 @@
 [env]
 # tree-sitter build fails with newer version of clang unless implicit-function-declaration is ignored
-CC_wasm32_wasi = { value = ".cargo/wasi-sdk-21.0/bin/clang -Wno-error=implicit-function-declaration", relative = true }
-AR_wasm32_wasi = { value = ".cargo/wasi-sdk-21.0/bin/ar", relative = true }
+CC_wasm32_wasi = { value = ".cargo/wasi-sdk-24.0/bin/clang -Wno-error=implicit-function-declaration", relative = true }
```

Not sure why rust is saying it can't find this file, but maybe it's no longer respecting the alias. Worth trying to skip it with this:

```suggestion
CC_wasm32_wasi = { value = ".cargo/wasi-sdk-24.0/bin/clang-18 -Wno-error=implicit-function-declaration", relative = true }
```

Worth noting that the previous version is using 17, so not sure if there's a fundamental incompatibility with the current rust version.